### DISCUSSION TITLE
reject the spf=none email

### DISCUSSION THREAD

**Discussion by sanpei3** [rspamd/rspamd#4811] [Q&A]
I want to reject the "spf=none" email, like Gmail with rspamd.

Please let me know the setting for rspamd.

**Comment by fatalbanana** [comment]
> I want to reject the "spf=none" email, like Gmail with rspamd.

You will find yourself rejecting perfectly good mail I suppose.

It was discussed previously [here](https://github.com/rspamd/rspamd/discussions/4641).

The [force_actions](https://rspamd.com/doc/modules/force_actions.html) module can be used to force an action if a particular symbol is present (such as `R_SPF_NA`).

**Reply by sanpei3** [reply]
Thank you for your information. I will check the previous discussion.
